### ISSUE TITLE
Problems installing Caiman toolbox in Windows 10 package based install

### ISSUE THREAD

**Issue by akhilaraman** [flatironinstitute/CaImAn#803]
For better support, please use the template below to submit your issue. When your issue gets resolved please remember to close it.

Sometimes errors while running CNMF occur during parallel processing which prevents the log to provide a meaningful error message. Please reproduce your error with setting `dview=None`.

If you need to upgrade CaImAn follow the instructions given in the [documentation](https://caiman.readthedocs.io/en/master/Installation.html#upgrading).

- **Tell us a bit about your setup**: 
 1. Operating system (Linux/macOS/Windows):  Windows 10
 
 2. Python version (3.x):  Python 3.7
 
 3. Working environment (Python IDE/Jupyter Notebook/other):   Anaconda/Spyder IDE
 
 4. Which of the demo scripts you're using for your analysis (if applicable): demo_caiman.basic.py
 
 5. CaImAn version*:  Latest version as of august 25,2020 https://github.com/flatironinstitute/CaImAn/
 
 6. CaImAn installation process (`pip install .`/`pip install -e .`/conda): 

I installed Caiman toolbox as per instructions in "Package based process" below, in windows 10.
https://caiman.readthedocs.io/en/master/Installation.html

The 3 steps below are successful:
conda create -n caiman  # caiman here refers to the name of the environment (you can pick any name you want)
conda activate caiman
conda install caiman -c conda-forge

- **Describe the issue that you are experiencing**

1.  The 2 install steps below FAIL. 
python caimanmanager.py check
python caimanmanager.py install

My user name in install directory has a space "FirstName Lastname", is that a problem?

 Attaching error log. 

2.   demo_caiman.basic.py   fails,

dims, T = get_file_size(file_name[0], var_name_hdf5=var_name_hdf5)
  File "C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\caiman\source_extraction\cnmf\utilities.py", line 1041, in get_file_size
    raise Exception('File not found!')

Exception: File not found!

Please help!

- **Copy error log below**

=====
(base) C:\Users\User 1>conda create -n caiman

WARNING: A space was detected in your requested environment path
'C:\Users\User 1\anaconda3\envs\caiman'
Spaces in paths can sometimes be problematic.
Collecting package metadata (current_repodata.json): done
Solving environment: done

Preparing transaction: done
Verifying transaction: done
Executing transaction: done

(base) C:\Users\User 1>conda activate caiman

(caiman) C:\Users\User 1>conda install caiman -c conda-forge

Executing transaction: - b'Enabling notebook extension jupyter-js-widgets/extension...\n      - Validating: ok\n'
\ DEBUG menuinst_win32:__init__(198): Menu: name: 'Anaconda${PY_VER} ${PLATFORM}', prefix: 'C:\Users\User 1\anaconda3\envs\caiman', env_name: 'caiman', mode: 'user', used_mode: 'user'
DEBUG menuinst_win32:create(323): Shortcut cmd is "C:\Users\User 1\anaconda3\pythonw.exe", args are ['"C:\\Users\\User 1\\anaconda3\\cwp.py"', '"C:\\Users\\User 1\\anaconda3\\envs\\caiman"', '"C:\\Users\\User 1\\anaconda3\\envs\\caiman\\pythonw.exe"', '"C:\\Users\\User 1\\anaconda3\\envs\\caiman\\Scripts\\spyder-script.py"']
DEBUG menuinst_win32:create(323): Shortcut cmd is "C:\Users\User 1\anaconda3\python.exe", args are ['"C:\\Users\\User 1\\anaconda3\\cwp.py"', '"C:\\Users\\User 1\\anaconda3\\envs\\caiman"', '"C:\\Users\\User 1\\anaconda3\\envs\\caiman\\python.exe"', '"C:\\Users\\User 1\\anaconda3\\envs\\caiman\\Scripts\\spyder-script.py"', '--reset']
done

C:\Users\User 1>set "MKL_NUM_THREADS="

C:\Users\User 1>set "OPENBLAS_NUM_THREADS="

C:\Users\User 1>set MKL_NUM_THREADS=1

C:\Users\User 1>set OPENBLAS_NUM_THREADS=1

(caiman) C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master>python caimanmanager.py check

C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
Traceback (most recent call last):
  File "caimanmanager.py", line 16, in <module>
    from caiman.paths import caiman_datadir
  File "C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\__init__.py", line 4, in <module>
    from .base.movies import movie, load, load_movie_chain
  File "C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\base\movies.py", line 62, in <module>
    from ..utils import visualization
  File "C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\utils\visualization.py", line 37, in <module>
    from ..summary_images import local_correlations
  File "C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\summary_images.py", line 23, in <module>
    from caiman.source_extraction.cnmf.pre_processing import get_noise_fft
  File "C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\source_extraction\cnmf\__init__.py", line 12, in <module>
    from . import oasis
ImportError: cannot import name 'oasis' from 'caiman.source_extraction.cnmf' (C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\source_extraction\cnmf\__init__.py)

(

(caiman) C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master>python caimanmanager.py install

C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\User 1\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
Traceback (most recent call last):
  File "caimanmanager.py", line 16, in <module>
    from caiman.paths import caiman_datadir
  File "C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\__init__.py", line 4, in <module>
    from .base.movies import movie, load, load_movie_chain
  File "C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\base\movies.py", line 62, in <module>
    from ..utils import visualization
  File "C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\utils\visualization.py", line 37, in <module>
    from ..summary_images import local_correlations
  File "C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\summary_images.py", line 23, in <module>
    from caiman.source_extraction.cnmf.pre_processing import get_noise_fft
  File "C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\source_extraction\cnmf\__init__.py", line 12, in <module>
    from . import oasis
ImportError: cannot import name 'oasis' from 'caiman.source_extraction.cnmf' (C:\Users\User 1\Downloads\CaImAn-master\CaImAn-master\caiman\source_extraction\cnmf\__init__.py)

============

- **If you're not reporting an error, type your message below**

**Comment by pgunn** [comment]
Hi, I have never tried installing Caiman under a directory with a space in it (in your case, "User 1". It might be worth creating a more normally-named account on your host to see if that helps.

**Comment by akhilaraman** [comment]
> 
> 
> Hi, I have never tried installing Caiman under a directory with a space in it (in your case, "User 1". It might be worth creating a more normally-named account on your host to see if that helps.

Thanks! I moved Caiman folder to  C:\Users\Public\Downloads>
and reinstalled it, removed old environment in "User 1" folder and
created new environment as below log.  

The 3 steps below are successful:
conda create -n caiman # caiman here refers to the name of the environment (you can pick any name you want)
conda activate caiman
conda install caiman -c conda-forge

  
    The 2 install steps below FAIL again!
    python caimanmanager.py check
    python caimanmanager.py install

attaching error log below. Any suggestions?

base) C:\Users\Public\Downloads>conda create -n caiman
WARNING: A conda environment already exists at 'C:\Users\Akhila Raman\anaconda3\envs\caiman'
Remove existing environment (y/[n])? y

Collecting package metadata (current_repodata.json): done
Solving environment: done

==> WARNING: A newer version of conda exists. <==
  current version: 4.8.3
  latest version: 4.8.4

Please update conda by running

    $ conda update -n base -c defaults conda


Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate caiman
#
# To deactivate an active environment, use
#
#     $ conda deactivate


(base) C:\Users\Public\Downloads>conda activate caiman

(caiman) C:\Users\Public\Downloads>conda install caiman -c conda-forge
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: done

Downloading and Extracting Packages
absl-py-0.10.0       | 169 KB    | ############################################################################ | 100%
intel-openmp-2020.2  | 1.6 MB    | ############################################################################ | 100%
cloudpickle-1.6.0    | 22 KB     | ############################################################################ | 100%
keyring-21.3.1       | 65 KB     | ############################################################################ | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: - b'Enabling notebook extension jupyter-js-widgets/extension...\n      - Validating: ok\n'
- DEBUG menuinst_win32:__init__(198): Menu: name: 'Anaconda${PY_VER} ${PLATFORM}', prefix: 'C:\Users\Akhila Raman\anaconda3\envs\caiman', env_name: 'caiman', mode: 'user', used_mode: 'user'
DEBUG menuinst_win32:create(323): Shortcut cmd is "C:\Users\Akhila Raman\anaconda3\pythonw.exe", args are ['"C:\\Users\\Akhila Raman\\anaconda3\\cwp.py"', '"C:\\Users\\Akhila Raman\\anaconda3\\envs\\caiman"', '"C:\\Users\\Akhila Raman\\anaconda3\\envs\\caiman\\pythonw.exe"', '"C:\\Users\\Akhila Raman\\anaconda3\\envs\\caiman\\Scripts\\spyder-script.py"']
\ DEBUG menuinst_win32:create(323): Shortcut cmd is "C:\Users\Akhila Raman\anaconda3\python.exe", args are ['"C:\\Users\\Akhila Raman\\anaconda3\\cwp.py"', '"C:\\Users\\Akhila Raman\\anaconda3\\envs\\caiman"', '"C:\\Users\\Akhila Raman\\anaconda3\\envs\\caiman\\python.exe"', '"C:\\Users\\Akhila Raman\\anaconda3\\envs\\caiman\\Scripts\\spyder-script.py"', '--reset']
done

C:\Users\Public\Downloads>set "MKL_NUM_THREADS="

C:\Users\Public\Downloads>set "OPENBLAS_NUM_THREADS="

C:\Users\Public\Downloads>set MKL_NUM_THREADS=1

C:\Users\Public\Downloads>set OPENBLAS_NUM_THREADS=1

(caiman) C:\Users\Public\Downloads>python caimanmanager.py check
python: can't open file 'caimanmanager.py': [Errno 2] No such file or directory

(caiman) C:\Users\Public\Downloads>cd CaImAn-master

(caiman) C:\Users\Public\Downloads\CaImAn-master>dir
 Volume in drive C has no label.
 Volume Serial Number is 1084-7F11

 Directory of C:\Users\Public\Downloads\CaImAn-master

08/25/2020  08:18 AM    <DIR>          .
08/25/2020  08:18 AM    <DIR>          ..
08/25/2020  08:18 AM             2,540 .gitignore
08/25/2020  08:18 AM               104 .style.yapf
08/25/2020  08:18 AM               652 .travis.yml
08/25/2020  08:18 AM               102 .yapfignore
08/25/2020  08:18 AM    <DIR>          bin
08/25/2020  12:56 PM    <DIR>          caiman
08/25/2020  08:18 AM            11,647 caimanmanager.py
08/25/2020  08:18 AM             2,918 CONTRIBUTING.md
08/25/2020  08:18 AM    <DIR>          demos
08/25/2020  08:18 AM               545 Dockerfile
08/25/2020  08:18 AM    <DIR>          docs
08/25/2020  08:18 AM               241 environment-minimal.yml
08/25/2020  08:18 AM               355 environment.yml
08/25/2020  08:18 AM    <DIR>          example_movies
08/25/2020  08:18 AM             1,154 ISSUE_TEMPLATE.md
08/25/2020  08:18 AM             3,812 Jenkinsfile
08/25/2020  08:18 AM            18,072 LICENSE.txt
08/25/2020  08:18 AM    <DIR>          model
08/25/2020  08:18 AM               153 mypy.ini
08/25/2020  08:18 AM             1,123 nbexec.py
08/25/2020  08:18 AM               210 pylint-all.sh
08/25/2020  08:18 AM               641 pylintrc
08/25/2020  08:18 AM             9,027 README.md
08/25/2020  08:18 AM               338 readthedocs.yml
08/25/2020  08:18 AM               244 requirements.txt
08/25/2020  08:18 AM               278 requirements_gpu.txt
08/25/2020  08:18 AM    <DIR>          sandbox
08/25/2020  08:18 AM             4,076 setup.py
08/25/2020  08:18 AM    <DIR>          SLURM
08/25/2020  08:18 AM    <DIR>          test
08/25/2020  08:18 AM    <DIR>          testdata
08/25/2020  08:18 AM             1,824 test_demos.sh
08/25/2020  08:18 AM    <DIR>          use_cases
08/25/2020  08:18 AM                 6 VERSION
              23 File(s)         60,062 bytes
              13 Dir(s)  669,340,364,800 bytes free

(caiman) C:\Users\Public\Downloads\CaImAn-master>python caimanmanager.py check
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
Traceback (most recent call last):
  File "caimanmanager.py", line 16, in <module>
    from caiman.paths import caiman_datadir
  File "C:\Users\Public\Downloads\CaImAn-master\caiman\__init__.py", line 4, in <module>
    from .base.movies import movie, load, load_movie_chain
  File "C:\Users\Public\Downloads\CaImAn-master\caiman\base\movies.py", line 62, in <module>
    from ..utils import visualization
  File "C:\Users\Public\Downloads\CaImAn-master\caiman\utils\visualization.py", line 37, in <module>
    from ..summary_images import local_correlations
  File "C:\Users\Public\Downloads\CaImAn-master\caiman\summary_images.py", line 23, in <module>
    from caiman.source_extraction.cnmf.pre_processing import get_noise_fft
  File "C:\Users\Public\Downloads\CaImAn-master\caiman\source_extraction\cnmf\__init__.py", line 12, in <module>
    from . import oasis
ImportError: cannot import name 'oasis' from 'caiman.source_extraction.cnmf' (C:\Users\Public\Downloads\CaImAn-master\caiman\source_extraction\cnmf\__init__.py)



(caiman) C:\Users\Public\Downloads\CaImAn-master>python caimanmanager.py install
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorflow\python\framework\dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\Akhila Raman\anaconda3\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
Traceback (most recent call last):
  File "caimanmanager.py", line 16, in <module>
    from caiman.paths import caiman_datadir
  File "C:\Users\Public\Downloads\CaImAn-master\caiman\__init__.py", line 4, in <module>
    from .base.movies import movie, load, load_movie_chain
  File "C:\Users\Public\Downloads\CaImAn-master\caiman\base\movies.py", line 62, in <module>
    from ..utils import visualization
  File "C:\Users\Public\Downloads\CaImAn-master\caiman\utils\visualization.py", line 37, in <module>
    from ..summary_images import local_correlations
  File "C:\Users\Public\Downloads\CaImAn-master\caiman\summary_images.py", line 23, in <module>
    from caiman.source_extraction.cnmf.pre_processing import get_noise_fft
  File "C:\Users\Public\Downloads\CaImAn-master\caiman\source_extraction\cnmf\__init__.py", line 12, in <module>
    from . import oasis
ImportError: cannot import name 'oasis' from 'caiman.source_extraction.cnmf' (C:\Users\Public\Downloads\CaImAn-master\caiman\source_extraction\cnmf\__init__.py)

(caiman) C:\Users\Public\Downloads\CaImAn-master>

**Comment by pgunn** [comment]
Hi,
I think what's happening is that you're mixing work from a download of the source tree (in the directory C:\Users\Public\Downloads\CaImAn-master ) and the packaged version. To use the packaged version, please remove the source tree directory entirely. If you want to run the demos, use caimanmanager to get you a working directory.

**Comment by akhilaraman** [comment]
Thanks! I removed "Caiman-master" folder away elsewhere.

To be clear, I should redo caiman install, as below? what instructions to "use caimanmanager to get you a working directory."?

In Anaconda3 prompt, I type
1. conda create -n caiman
2. conda activate caiman
3. conda install caiman -c conda-forge

Do you mean these steps below? It is complaining " caimanmanager.py" not found!

4. python caimanmanager.py check
5. python caimanmanager.py install

**Comment by pgunn** [comment]
There should be a copy of caimanmanager.py installed under your anaconda3\envs\caiman\bin\

(the path should be fairly close to that)
so do:

`python anaconda3\envs\caiman\bin\caimanmanager.py install`

(adjust path as necessary)

That should get you a `caiman_data` directory.

**Comment by akhilaraman** [comment]
Thank you, Pat! I have got it running as per your instruction.

I had to uninstall Anaconda3 and reinstall it in a  folder name without spaces, reinstall caiman and
I was able to get caiman_data folder installed and can run demo_caiman_basic.py in Spyder IDE!

There was a "Future Warning" while installing caiman_data directory as below, but it doesn't affect the demo.

C:\Users\Public\Anaconda\envs\caiman\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])

Installed C:\Users\User 1\caiman_data
